cli/lib/kontena/cli/grids/update_command.rb

```diff
@@ -8,10 +8,13 @@ class UpdateCommand < Kontena::Command
     parameter "NAME", "Grid name"
     option "--statsd-server", "STATSD_SERVER", "Statsd server address (host:port)"
     option "--default-affinity", "[AFFINITY]", "Default affinity rule for the grid", multivalued: true
+    option "--log-driver", "LOG_DRIVER", "Set grid wide log driver"
+    option "--log-opt", "[LOG_OPT]", "Set log options (key=value)", multivalued: true
```

Comment:
Ideally these same options would also exist on `kontena grid create`.... but it's not just the `--log` options that suffer from this.